.github/workflows/test.yml

```diff
@@ -66,3 +66,31 @@ jobs:
       # Cleanup cached paths
       - run: tarantoolctl rocks remove cartridge
       - run: tarantoolctl rocks remove ddl
+  benchmark:
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool: ['1.10', '2.5', '2.6', '2.7']
+        coveralls: [false]
+        include:
+          - tarantool: '2.8'
+            coveralls: true
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      # Setup luatest
+      - name: Cache rocks
+        uses: actions/cache@v2
+        id: cache-rocks
+        with:
+          path: .rocks/
+          key: cache-rocks-${{ matrix.runs-on }}-06
```

Comment:
The key is the same as in the job above, but stored `.rocks` content is different. I imagine the following situation: we store `.rocks` with luatest here and then next 'test' job will find the cache and use it. So it'll skip installation of luacheck, luacov and other modules and will fail.

Let's use another key.

Reply:
BTW, as I see from the benchmark code, it does not depend on luatest. Is it so?

Reply:
depends from local helper = require('test.helper') -> local helpers = table.copy(require('luatest').helpers)

Reply:
fixed